cmd/nerdctl/load.go

```diff
@@ -38,6 +38,10 @@ var loadCommand = &cli.Command{
 			Aliases: []string{"i"},
 			Usage:   "Read from tar archive file, instead of STDIN",
 		},
+		&cli.BoolFlag{
+			Name:  "all-platforms",
+			Usage: "Imports content for all platforms, false by default",
```

Comment:
"false by default" can be removed, as `urfave/cli/v2` appends `(default: false)` automatically